multicluster/controllers/multicluster/stale_controller.go

```diff
@@ -434,8 +434,8 @@ func (c *StaleResCleanupController) processNextWorkItem() bool {
 		return false
 	}
 	defer c.queue.Done(key)
-
-	err := c.cleanup()
+	ctx := context.Background()
+	err := c.cleanup(ctx)
```

Comment:
I think it should pass the context from `Run`, the worker should be started with `UntilWithContext`.

https://github.com/kubernetes/kubernetes/blob/34abd36e88e50d913af9413f9353515deb7f09d4/pkg/controller/daemon/daemon_controller.go#L299

Reply:
Updated, thanks.